HIP 93: Addition of Wi-Fi Access Points to the Helium Mobile SubDAO

meowshka opened the pull request.

waveform06 commented:
Comments - really about making the HIP easier to understand for those not familiar with 

HIP mentions "subscribers" and "users of the Helium Mobile Network", are they the same?
Can the "users of the Helium Mobile Network" be defined? Eg devices with an eSIM with a contract that allows roaming use on the Helium CBRS/Wifi networks. (Reason: Other Wifi subnetworks may offer subscriptions for non-eSIM devices to access the Wifi network.) eg Can I connect my Printer or my SmartHome devices to this network will be common questions if not defined.

The HIP does not mention Hotspots at all, clarification is needed that these devices are standalone Hotspots (Gateway + Radio) or Radios that require connection to an existing Hotspot?

Where it says "tested, and approved in accordance with security guidelines issued by the Helium Foundation via dedicated HIPs." do you mean "tested, and approved in accordance with relevant HIPs issued by the Manufacturer Compliance Committee"?

Onboarding indoors: Mention is made of "using an app in a mobile device" and "mobile phone running the app". Is device used as it may include additional devices that are not "phones" ?
Is the "Helium Mobile app" actually the Helium Hotspot App?, or is a new Maker/Mapping app in development?
Onboarding outdoors: Is the app needed to perform the Outdoor onboarding process, but no Wifi/GPS needed from the device running the app?

3.1.3 = Indoor Hex capping and and 3.2.4 = Outdoor Hex capping ?
Confirm if this is the "coverage_claim_time" part of the reward algorithms described in HIP74?

3.4.1 mentions section 2.1.1.2 which doesnt exist, is that 3.2.1?

Unresolved questions.
Location assert fee has not been mentioned or defined.
It will help the discussion of onboard/location assert fees if estimated prices of Indoor and Outdoor devices are available.
Are there any off-chain location assertion/verification fees for the Outdoor Wi-Fi Professional Installer

meowshka commented:
> Comments - really about making the HIP easier to understand for those not familiar with
> 
> HIP mentions "subscribers" and "users of the Helium Mobile Network", are they the same? Can the "users of the Helium Mobile Network" be defined? Eg devices with an eSIM with a contract that allows roaming use on the Helium CBRS/Wifi networks. (Reason: Other Wifi subnetworks may offer subscriptions for non-eSIM devices to access the Wifi network.) eg Can I connect my Printer or my SmartHome devices to this network will be common questions if not defined.

- Added definition of the Subscriber as a reference to HIP53 which defines who are classified as Subscribers.

> The HIP does not mention Hotspots at all, clarification is needed that these devices are standalone Hotspots (Gateway + Radio) or Radios that require connection to an existing Hotspot?

- Added "Indoor and Outdoor Wi-Fi access points are standalone devices that include a radio and a gateway and do not require an external gateway for their operation." to 1.1. and rename WiFi Radios to WiFi Access Points to minimize further confusion.

> Where it says "tested, and approved in accordance with security guidelines issued by the Helium Foundation via dedicated HIPs." do you mean "tested, and approved in accordance with relevant HIPs issued by the Manufacturer Compliance Committee"?
- Updated to ref "Manufacturer Compliance Committee".

> Onboarding indoors: Mention is made of "using an app in a mobile device" and "mobile phone running the app". Is device used as it may include additional devices that are not "phones" ? Is the "Helium Mobile app" actually the Helium Hotspot App?, or is a new Maker/Mapping app in development? Onboarding outdoors: Is the app needed to perform the Outdoor onboarding process, but no Wifi/GPS needed from the device running the app?
-  No, Helium Mobile app is a standalone app. 
- GPS data will be pulled from outdoor device, not the app.

> 3.1.3 = Indoor Hex capping and and 3.2.4 = Outdoor Hex capping ? Confirm if this is the "coverage_claim_time" part of the reward algorithms described in HIP74?
- Added ref to HIP74 for clarity.
 
> 3.4.1 mentions section 2.1.1.2 which doesnt exist, is that 3.2.1?
- Fixed this and other references.

> Unresolved questions. Location assert fee has not been mentioned or defined. It will help the discussion of onboard/location assert fees if estimated prices of Indoor and Outdoor devices are available. Are there any off-chain location assertion/verification fees for the Outdoor Wi-Fi Professional Installer
- There are no Professional Installer fees anticipated. Onboarding and location assertions fees will be addressed in the separate HIP.
